instrumentation/internal/internal-reflection/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/internal/reflection/ReflectionInstrumentationModule.java

```diff
@@ -26,12 +30,20 @@ public boolean defaultEnabled(ConfigProperties config) {
   }
 
   @Override
-  public boolean isIndyModule() {
-    return false;
+  public List<TypeInstrumentation> typeInstrumentations() {
+    return asList(new ClassInstrumentation(), new ReflectionInstrumentation());
   }
 
   @Override
-  public List<TypeInstrumentation> typeInstrumentations() {
-    return asList(new ClassInstrumentation(), new ReflectionInstrumentation());
+  public String getModuleGroup() {
+    return "internal-reflection";
```

Comment:
Is the module group here necessary? It seems to be the only module in that group, in that case there is no need to override this method

Reply:
Good point, it's not required here. I'll wait for the tests to complete in CI and remove it once they pass.